### DISCUSSION TITLE
Build local failed

### DISCUSSION THREAD

**Discussion by leanhvu1989** [vendurehq/vendure#2106] [Q&A]
Hi all, I follow the guide https://github.com/vendure-ecommerce/vendure
I'm stuck at step **3. Build all packages**
<img width="1098" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/25602099/228506463-f655d0f7-5096-429b-9d7f-eb9a11633cd3.png">

I really appreciate any help. Thanks

**Comment by michaelbromley** [comment]
Hi,
can you provide:

- node version
- yarn version

**Reply by leanhvu1989** [reply]
> Hi, can you provide:
> 
> * node version
> * yarn version

I'm using:
- Node: **v16.13.0**
- Yarn: **1.22.19**

**Reply by leanhvu1989** [reply]
hi @michaelbromley , when I run yarn to install dependencies, I got some error relate to sqlite3.
So, I must install sqlite by `yarn add "sqlite3@^5" --ignore-workspace-root-check`, then I run `yarn` and `yarn bootstrap ` normally.
So, do you have any idea to solve this problem? Tyvm

**Comment by leanhvu1989** [comment]
still waiting for any help. Thanks

**Reply by michaelbromley** [reply]
I'm sorry, I don't know the cause of this issue. By the way, are you attempting to build the monorepo because you wish to contribute some code to the project?

**Reply by leanhvu1989** [reply]
yes, I want to contribute to the project. That's why I need to run all at local.

**Reply by michaelbromley** [reply]
Check this thread if you didn't already see it - it might point you in the direction of a solution: https://github.com/yarnpkg/yarn/issues/4492
